GVRf/Framework/framework/src/main/java/org/gearvrf/VrapiActivityHandler.java

```diff
@@ -150,19 +152,27 @@ public void onSetScript() {
         holder.setFormat(PixelFormat.TRANSLUCENT);
 
         final VrAppSettings appSettings = mActivity.getAppSettings();
-        final int framebufferHeight = appSettings.getFramebufferPixelsHigh();
-        final int framebufferWidth = appSettings.getFramebufferPixelsWide();
-        if (-1 != framebufferHeight && -1 != framebufferWidth && screenWidthPixels != framebufferWidth
-                && screenHeightPixels != framebufferHeight) {
-            Log.v(TAG, "--- window configuration ---");
-            Log.v(TAG, "--- width: %d", framebufferWidth);
-            Log.v(TAG, "--- height: %d", framebufferHeight);
-            //a different resolution of the native window requested 
-            holder.setFixedSize(framebufferWidth, framebufferHeight);
-            Log.v(TAG, "----------------------------");
+        mScreenDimensions = new Vector2f(appSettings.getFramebufferPixelsHigh(), appSettings.getFramebufferPixelsWide());
```

Comment:
should we be using the DIsplayMetrics values instead of the framebuffer? Say someone sets the values to 100x100 in using gvr.xml then it would only create a view of 100x100.